docs/content/en/docs/reference/metrics.md

```diff
@@ -0,0 +1,422 @@
+---
+title: "Metrics Reference"
+description: >
+  This reference documents Prometheus metrics exposed by Tetragon.
```

Comment:
The other descriptions explain a bit how this doc is generated, with:
- This reference is generated from the Tetragon Helm chart values.
- This reference is generated from the protocol buffer specification and documents the gRPC API of Tetragon.

maybe we can add a hint as well that could interesting?